docs/getting-started-guides/kubeadm.md

```diff
@@ -37,20 +37,22 @@ You will install the following packages on all the machines:
 * `kubelet`: the most core component of Kubernetes.
   It runs on all of the machines in your cluster and does things like starting pods and containers.
 * `kubectl`: the command to control the cluster once it's running.
-  You will only use this on the master.
+  You will only need this on the master, but it can be useful to have on the other nodes as well.
 * `kubeadm`: the command to bootstrap the cluster.
 
 For each host in turn:
 
 * SSH into the machine and become `root` if you are not already (for example, run `sudo su -`).
-* If the machine is running Ubuntu 16.04, run:
+* If the machine is running Ubuntu 16.04 or HypriotOS, run:
 
       # curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key add -
       # cat <<EOF > /etc/apt/sources.list.d/kubernetes.list
       deb http://apt.kubernetes.io/ kubernetes-xenial main
       EOF
       # apt-get update
-      # apt-get install -y docker.io kubelet kubeadm kubectl kubernetes-cni
+      # # Install docker if you don't have it already. Version 1.11.2 is recommended, but 1.10.3 and 1.12.1 are known to work as well.
```

Comment:
Not sure if this helps the user here, may be we can have these extra version number in footnotes?